tests/utils/LHNTestUtils.tsx

```diff
@@ -129,13 +130,23 @@ let lastFakeReportActionID = 0;
 function getFakeReport(participantAccountIDs = [1, 2], millisecondsInThePast = 0, isUnread = false): Report {
     const lastVisibleActionCreated = DateUtils.getDBTime(Date.now() - millisecondsInThePast);
 
+    const participants: Record<number, Participant> = {};
+
+    participantAccountIDs.forEach((id) => {
+        participants[id] = {
+            hidden: false,
+            role: id === 1 ? 'admin' : 'member',
```

Comment:
Do we need to set the role here?

Reply:
This would affect other tests that uses this util function. Add the admin participant if needed only on the group test file